spec/services/big_blue_button_api_spec.rb

```diff
@@ -4,13 +4,15 @@
 require 'bigbluebutton_api'
 
 describe BigBlueButtonApi, type: :service do
+  let(:bbb_server) { instance_double(BigBlueButton::BigBlueButtonApi) }
```

Comment:
What's the purpose of this?

Reply:
It's intended for instance mocking of the `BigBlueButton::BigBlueButtonApi` class.
I need to mock the gem instances to ensure an isolated, fast and safe tests otherwise an actual BBB API call will be issued and may fail the tests.
The `allow_any_instance_of` isn't recommended any more.
After giving it a second look, do you like me to keep include it only for start session context since it invalidates the `BigBlueButton client initialized once per request` example?

Reply:
DONE,